### PR TITLE
pull request

### PR DESCRIPTION
**Pull request by zee786zee**
Thanks for your contribution to [Apache Commons](https://commons.apache.org/)! Your help is appreciated!

Before you push a pull request, review this list:

- [ ] Read the [contribution guidelines](CONTRIBUTING.md) for this project.
- [ ] Run a successful build using the default [Maven](https://maven.apache.org/) goal with `mvn`; that's `mvn` on the command line by itself.
- [ ] Write unit tests that match behavioral changes, where the tests fail if the changes to the runtime are not applied. This may not always be possible but is a best-practice.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Each commit in the pull request should have a meaningful subject line and body. Note that commits might be squashed by a maintainer on merge.

### CODE REVIEW

**Comment by zee786zee** [comment]
dne

File: .run/Install without tests.run.xml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+/*
```
Comment (garydgregory) [inline .run/Install without tests.run.xml:1]: I'm not sure what this file is of why it's needed. Please remove it.

File: pom.xml [hunk 2]
```diff
@@ -32,10 +32,10 @@ limitations under the License.
   <name>Apache Commons Codec</name>
   <inceptionYear>2002</inceptionYear>
   <description>
-     The Apache Commons Codec component contains encoders and decoders for
```
Comment (garydgregory) [inline pom.xml:35]: Please reduce the edits to the bare minimum, for example, no style changes.

File: pom.xml [hunk 3]
```diff
@@ -101,22 +110,29 @@ limitations under the License.
     <commons.distSvnStagingUrl>scm:svn:https://dist.apache.org/repos/dist/dev/commons/${commons.componentid}</commons.distSvnStagingUrl>
     <project.build.outputTimestamp>2024-07-15T22:27:42Z</project.build.outputTimestamp>
     <commons.jacoco.version>0.8.12</commons.jacoco.version>
-    <!-- Temp for SUREFIRE-2253 -->
-    <commons.surefire.version>3.2.5</commons.surefire.version>
-    <!-- JaCoCo: Don't make code coverage worse than: -->
     <jacoco.skip>false</jacoco.skip>
     <commons.jacoco.haltOnFailure>true</commons.jacoco.haltOnFailure>
-    <commons.jacoco.classRatio>0.92</commons.jacoco.classRatio>
```
Comment (garydgregory) [inline pom.xml:109]: Do NOT edit this section.

**Review by garydgregory — CHANGES_REQUESTED** [review]
-1 as is, please read the pull request template and see my comments.

**Comment by garydgregory** [comment]
Closing: No reply.

**Pull request closed without merging**